packages/@react-aria/overlays/docs/DismissButton.mdx

```diff
@@ -12,7 +12,7 @@ export default Layout;
 
 import docs from 'docs:@react-aria/overlays';
 import {HeaderInfo, PropTable} from '@react-spectrum/docs';
-import packageData from '../package.json';
+import packageData from '@react-aria/overlays/package.json';
```

Comment:
We need to make sure we always load the package.json this way from now on. When building for production, the code and docs aren't in the same file structure so relative paths won't work.